pkg/tinydtls/Makefile.dep

```diff
@@ -1,9 +1,41 @@
 ifneq (,$(filter tinydtls,$(USEPKG)))
   USEMODULE += tinydtls
+
+  # Mandatory for tinyDTLS
+  CFLAGS += -DDTLSv12 -DWITH_SHA256
```

Comment:
IMHO all of these CFLAGS modification should go into `pkg/tinydtls/Makefile.include` or even in the `pkg/tinydtls/Makefile`, the .dep should only describe module dependencies.